metadata/org.apache.curator/curator-framework/5.5.0/reflect-config.json

```diff
@@ -0,0 +1,65 @@
+[
```

Comment:
This file doesn't contain any metadata related to the curator framework. The `java` related metadata like the following one
```
  {
    "condition":{"typeReachable":"org.apache.curator.framework.imps.NamespaceWatcher"},
    "name":"java.lang.Object",
    "methods":[{"name":"toString","parameterTypes":[] }]
  }
```
don't have to be added to the reflect-config.json file even if those were generated by the native agent. The same applies for `sun.misc` and `sun.security` metadata. Shaded metadata for some other library should not also be part of curator framework metadata. You can filter out those from native agent results using `extraFilterPath` in the `build.gradle` file like here
```
graalvmNative {
    agent {
        defaultMode = "conditional"
        modes {
            conditional {
                userCodeFilterPath = "user-code-filter.json"
                extraFilterPath = "extra-filter.json"
            }
        }
    }
}
```
Also the CuratorFrameworkTest can be successfully executed in a native image without any of these metadata entries. It just requires zookeeper metadata.

Reply:
- Solve.  Due to the discovery of https://github.com/oracle/graalvm-reachability-metadata/pull/381#discussion_r1333368920 , this PR will now directly fix https://github.com/oracle/graalvm-reachability-metadata/issues/163  .